src/Components/Publishing/Video/VideoInfoBlock.tsx

```diff
@@ -5,40 +5,44 @@ import { formatTime } from "../Constants"
 import { Fonts } from "../Fonts"
 
 interface Props {
+  editTitle?: any
   media?: any
-  seriesTitle?: string
+  tag?: string
+  title?: string
 }
 
 export class VideoInfoBlock extends Component<Props, null> {
   render() {
     const {
+      editTitle,
       media,
-      seriesTitle,
+      tag,
+      title
```

Comment:
Tag is a bit of a confusing name here because it is not literally a tag, although that is something that exists on articles. Maybe SuperTitle or SubTitle?

Reply:
Good suggestion, thanks!